README.md

```diff
@@ -117,6 +117,8 @@ Under the hood it uses [Karma test runner](http://karma-runner.github.io/) to al
 
 By default Sagui uses [PhantomJS](http://phantomjs.org/) to run the tests headlessly. To **speed up installing the dependencies** (`npm install`) it is advisable to have PhantomJS [installed globally](https://github.com/Medium/phantomjs#using-phantomjs-from-disk) in the machine.
 
+To open the tests in a browser (or in multiple browsers!), simply follow the link Karma outputs when you start running the script `test:unit:watch`. Running them in a browser allows you to set breakpoints and debug your code properly. Note watch mode is necessary, else tests will stop running when finished.
```

Comment:
Maybe add the URL here? "...simply follow the link Karma outbputs (http://localhost:9876/)..."